src/renderer/components/CountrySelect.vue

```
@@ -84,6 +89,17 @@
 
         this.countriesList = getCountryListFromProposals(proposals)
       })
+
+      this.rendererCommunication.onConnectionRequest((proposal) => {
+        let selectedCountry = this.countriesList.find((country) => country.id === proposal.providerId)
+
+        if (!selectedCountry) {
```

Should not You show error to user on this case? Currently silently skipping user's desire

---

@Waldz Not sure how to handle it though. User selects tray, but if country no longer exists in the dropdown there's not much we can do. If the country is no longer available, user will not get connected anyway and will be shown an error, so adding another error isn't exactly an option.

---

Fixed. If the country isn't found after clicking tray, the dropdown is reset into "Choose country"